samples/caf_sensor_manager/remote/boards/nrf54h20pdk_nrf54h20_cpuppr.overlay

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+ / {
+	reserved-memory {
+		ram3x_agg_area0: memory@2fc05000 {
+			reg = <0x2fc05000 0x600>;
+		};
```

Comment:
compatible here?

Reply:
tagging @57300 as he's the guy behind upstream memory map

Reply:
What upstream memory map has to do with it? This is part of the memory that application reserves for its internal purposes.

Reply:
No compatible is needed. I'm only concerned about overlap with the PPR RAM region, which could become a problem if the sample's RAM usage increases.

I suggest packing it into the `cpuppr_ram3x_region`:

```
/ {
	chosen {
		zephyr,sram = &cpuppr_code_data;
	};
};

&cpuppr_ram3x_region {
	#address-cells = <1>;
	#size-cells = <1>;
	ranges = <0x0 0x2fc00000 0x7000>;

	cpuppr_code_data: memory@0 {
		reg = <0x0 0x6a00>;
	};

	ram3x_agg_area0: memory@6a00 {
		reg = <0x6a00 0x600>;
	};
};
```
(This will be a little easier in FP1)